offchain/eth-input-reader/src/config.rs

```diff
@@ -19,12 +19,10 @@ use types::deployment_files::{
     rollups_deployment::{RollupsDeployment, RollupsDeploymentJson},
 };
 
-use crate::auth::{AuthConfig, AuthEnvCLIConfig, AuthError};
-
 #[derive(Clone, Parser)]
 #[command(name = "rd_config")]
-#[command(about = "Configuration for rollups dispatcher")]
-pub struct DispatcherEnvCLIConfig {
+#[command(about = "Configuration for rollups eth-input-reader")]
+pub struct EthInputReaderEnvCLIConfig {
```

Comment:
We still have the `TxEnvCLIConfig`  but the eth-input-reader doesn't send transactions. This seems wrong.

Reply:
@GMKrieger FYI